### ISSUE TITLE
fix tests

### ISSUE THREAD

**Issue by nikonikoniko** [dw-innovation/dw-kid2-graph-frontend-prototype#129]
Our refactor has broken the tests.  I assume the imports are silly.

We can also set up the tests to:
-- test the services and their construction.  this would involve mocking them.

**Comment by tlmn** [comment]
Yes, I think it broke when we did the type refactor and TS didn't complain since it's not checking those testing files. Will fix it.
